src/renderer/components/dialogs/MuteChat.tsx

```diff
@@ -1,20 +1,20 @@
-import { SmallSelectDialog, SelectDialogOption } from './DeltaDialog'
 import React from 'react'
+
+import { SmallSelectDialog, SelectDialogOption } from './DeltaDialog'
 import { Timespans } from '../../../shared/constants'
 import { selectedAccountId } from '../../ScreenController'
 import { BackendRemote, Type } from '../../backend-com'
+import { useTranslationFunction } from '../../hooks/useTranslationFunction'
 
 export default function MuteChat({
-  isOpen,
   onClose,
   chatId,
 }: {
-  isOpen: boolean
   onClose: () => void
   chatId: number
 }) {
```

Comment:
Should not it be part of `DialogProps`?

Reply:
it is not needed anymore because of this pr

Reply:
> Should not it be part of `DialogProps`?

`onClose` should be taken from `DialogProps` though 👍🏻 I fix that!